examples/08_04_concurrent_tcp_server/src/main.rs

```diff
@@ -0,0 +1,31 @@
+// ANCHOR: main_func
+use async_std::net::TcpListener;
+use async_std::net::TcpStream;
+use futures::stream::StreamExt;
+
+#[async_std::main]
+async fn main() {
+    let listener = TcpListener::bind("127.0.0.1:7878").await.unwrap();
+    listener
+        .incoming()
+        .for_each_concurrent(/* limit */ None, |stream| async move {
```

Comment:
nit: Perhaps use `tcp_stream` here in order to avoid confusion with the TcpListener stream that we talk about elsewhere. (Since this is a case of nested streams)